winforms/src/toga_winforms/app.py

```diff
@@ -61,6 +59,17 @@ def create(self):
         self.native = WinForms.Application
         self.app_context = WinForms.ApplicationContext()
 
+        # Get the app dispatcher
+        # https://github.com/proneon267/dotnet-core-redist-lists/blob/098fd735aeb41313e4e1b20829a911d7162b20b5/AssemblyList_4_client.xml#L91
```

Comment:
This isn't a reference we're going to want to keep in the codebase. A reference to a 2 day old, unnofficial repository with no indication of the provenance of the data contained isn't something that is helpful to future code explorers.

What *would* be helpful is a description of what is being loaded, and why (and why it's not being done with a more traditional `clr.AddReference()` call.

Reply:
I will add a better description. I couldn't find the assembly name list along with their correct parameters anywhere on the internet. So, I had to extract them to found the correct Assembly name string. Just so I don't lose them in the future, I had uploaded them.